libs/dh/metering-point/feature-overview/src/lib/child-metering-point-tab-content/dh-child-metering-point-tab-content.component.scss

```diff
@@ -39,7 +31,3 @@
 .multi-line-cell {
```

Comment:
This class also sounds like being much related to the styling of the table, consider moving this as well.